docs/proposals/20220330-topology-mutation-hook.md

```diff
@@ -219,6 +227,14 @@ Mitigations:
 * External Patch Extension developers should ensure fast responses under all circumstances.
 * Cluster operators can set a timeout on the RuntimeExtensionConfiguration to ensure Cluster topology reconciliation for all Clusters is not slowed down by one slow External Patch Extension. This only helps if the slow External Patch Extension is not used for all Clusters.
 
+#### Clashing external variable definitions
+Variable definitions supplied externally by an External Patch Extension though a Variable Discovery Hook can change when the definition in the External Patch Extension changes. This can lead to a clash where variables that previously had the same name and definition no longer have the same definition.
```

Comment:
```suggestion
Variable definitions supplied externally by an External Patch Extension through a Variable Discovery Hook can change when the definition in the External Patch Extension changes. This can lead to a clash where variables that previously had the same name and definition no longer have the same definition.
```